Lesson31/demo-app/merve2/src/shared-components/typography/BodyS/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import 'src/shared-components/typography/BodyS/styles.css';
+
+const BodyS = () => {
```

Comment:
This component wouldn't be useful as it's not possible to customize the text. Take 'text' value as prop, then use it inside the component:
```
const BodyS = ({text}) => {
  return <p className='body-s'>{text}</p>
}
```